### PR TITLE
(MCO-811) Fixnum is deprecated, use Integer

### PR DESCRIPTION
**Pull request by ripienaar**
Ruby 2.4 deprecated Fixnum and Bignum, Integer should now be used

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,6 +9,10 @@ else
   gem 'systemu'
 end
 
+if RUBY_VERSION =~ /^2\.4/
+  gem 'json', '~> 2.1.0'
```
Comment (MikaelSmith) [inline Gemfile]: Why is this needed?

Comment (ripienaar) [inline Gemfile]: Old gem also not compatible with new ruby. It's native C code references the C version of Fixnum which doesn't compile anymore
